pyproject.toml

```diff
@@ -58,6 +58,7 @@ beautifulsoup4 = "^4.9.1"
 requests = "^2.23.0"
 docx2txt = {version = "^0.8", optional = true}
 pypdf = {version = "^3.3.0", optional = true}
+aiohttp = "^3.8.4"
```

Comment:
It would be better to make this optional, because otherwise this would introduce a new dependency and increase the size of the package. Users will not like this (myself included). Generally I'm against adding dependencies, but if its optional, then its ok, so IMO just make it optional and add an `extra_requires: async` for it

Reply:
Ok adding to checklist for the task.